champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/assassin/axe/Leap.java

```diff
@@ -41,9 +41,13 @@ public String getName() {
     @Override
     public String[] getDescription(int level) {
         return new String[]{
-                "Right click with a axe to activate.",
+                "Right click with an Axe to activate",
+                "",
+                "Take a great leap forward",
+                "",
+                "Using near a wall will activate a wall-kick",
```

Comment:
Is this actually how leap here is implemented? I couldn't figure it out and in testing it did not appear to work this way.

Reply:
yeah it has a wall kick functionality